README.md

```diff
@@ -28,12 +28,12 @@ Both above authentication methods will return a JWT that contains the user's ID
 
 Example: Get token with User Authentication
 ```
-curl -X POST -F 'username=admin' -F 'password=password' http://local.dev/wp-json/tide/v2/auth
+curl -X POST -F 'username=admin' -F 'password=password' http://local.dev/wp-json/tide/v1/auth
 ```
 
 Example: Get token with API Key and API Secret
 ```
-curl -X POST -F 'api_key=1guJ0omfNkzoXsUu6NRp9rwCr' -F 'api_secret=X)WxT3Wp&NPHPWFFxRGDioP1UUVNuBfH' http://local.dev/wp-json/tide/v2/auth
+curl -X POST -F 'api_key=1guJ0omfNkzoXsUu6NRp9rwCr' -F 'api_secret=X)WxT3Wp&NPHPWFFxRGDioP1UUVNuBfH' http://local.dev/api/tide/v2/auth
```

Comment:
Let's change `local.dev` to `[SITE]` like below.

Reply:
@rheinardkorf Also this should be `v1` not v2`.

Reply:
good catch